<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="abc-PC" name="QueryAction" tests="6" failures="1" timestamp="21 Nov 2017 10:44:13 GMT" time="170.684" errors="0">
  <testcase name="loginWithValidCredentials" time="140.341" classname="tests.QueryActionTest"/>
  <testcase name="verifyValidationMessage" time="9.252" classname="tests.QueryActionTest"/>
  <testcase name="statusVerificationAfterClickOnCloseBtn" time="2.774" classname="tests.QueryActionTest"/>
  <testcase name="sendQuery" time="5.06" classname="tests.QueryActionTest"/>
  <testcase name="verifyQueryLink" time="0.192" classname="tests.QueryActionTest"/>
  <testcase name="verifyStatusLink" time="0.107" classname="tests.QueryActionTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
expected [185152] but found [1]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [185152] but found [1]
	at steps.ActiveProjectsSteps.verifyingStatusLnkSteps(ActiveProjectsSteps.java:105)
	at tests.QueryActionTest.verifyStatusLink(QueryActionTest.java:70)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyStatusLink -->
</testsuite> <!-- QueryAction -->
